Backpack/Chip/Classes/BPKChip.m

```diff
@@ -264,6 +292,20 @@ - (void)updateTitle {
     [self.titleLabel setTextColor:self.contentColor];
 }
 
+- (void)updateOutline {
```

Comment:
This needs to be updated when the trait colleciton changes otherwise it will not correctly react when changing from dark mode to light mode or vice versa

Reply:
Oh right! totally forgot that!

Reply:
This is already handled because I'm calling this from updateStyle method.